patches/chrome-BUILD.gn.patch

```diff
@@ -1,19 +1,17 @@
 diff --git a/chrome/BUILD.gn b/chrome/BUILD.gn
-index 5eeb0d0c2598bbcff4981dcb6f43cae82ec37c99..d20263d25250da69f53b9b7e5322e019a7bda879 100644
+index 5eeb0d0c2598bbcff4981dcb6f43cae82ec37c99..9755362d12402497e1c7d53e938a32275f0bc1f7 100644
 --- a/chrome/BUILD.gn
 +++ b/chrome/BUILD.gn
-@@ -165,6 +165,10 @@ if (!is_android && !is_mac) {
+@@ -165,6 +165,8 @@ if (!is_android && !is_mac) {
          "common/crash_keys.cc",
          "common/crash_keys.h",
        ]
-+      if (brave_chromium_build) {
-+        sources -= [ "app/chrome_exe.rc", ]
-+        sources += [ "//brave/app/brave_exe.rc", ]
-+      }
++      sources -= [ "app/chrome_exe.rc", ]
++      sources += [ "//brave/app/brave_exe.rc", ]
```

Comment:
Can collapse into a single line.

Reply:
Hmmm... how do we do that? Note that one it's an addition and the other one is a removal :)

Reply:
Ah, ok... you mean putting the two statements in the same line, not merging the two lists. Ok